api/wallet_test.go

```diff
@@ -456,6 +457,68 @@ func TestIntegrationWalletSweepSeedPOST(t *testing.T) {
 	}
 }
 
+// TestIntegrationWalletLoadSeedPOST probes the POST call to
+// /wallet/seed.
+func TestIntegrationWalletLoadSeedPOST(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
+	// Create a wallet.
+	key := crypto.TwofishKey(crypto.HashObject("password"))
+	st, err := assembleServerTester(key, build.TempDir("api", t.Name()))
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Mine blocks until the wallet has confirmed money.
+	for i := types.BlockHeight(0); i <= types.MaturityDelay; i++ {
+		st.miner.AddBlock()
+	}
+
+	// Create a wallet to load coins from.
+	key2 := crypto.GenerateTwofishKey()
+	w2, err := wallet.New(st.cs, st.tpool, filepath.Join(st.dir, "wallet2"))
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = w2.Encrypt(key2)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = w2.Unlock(key2)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Mine coins into the second wallet.
+	m, err := miner.New(st.cs, st.tpool, w2, filepath.Join(st.dir, "miner2"))
+	if err != nil {
+		t.Fatal(err)
+	}
+	for i := types.BlockHeight(0); i <= types.MaturityDelay; i++ {
+		m.AddBlock()
+	}
+
+	// Record starting balances.
+	oldBal, _, _ := st.wallet.ConfirmedBalance()
+	w2bal, _, _ := w2.ConfirmedBalance()
```

Comment:
should drop a sanity check here that `w2bal` != 0. Otherwise whole test looks good.